sdk/avs/Azure.ResourceManager.Avs/CHANGELOG.md

```diff
@@ -1,16 +1,15 @@
 # Release History
 
-## 1.4.0-beta.1 (Unreleased)
+## 1.4.0-beta.1 (2024-10-28)
 
 ### Features Added
 
 - Exposed `JsonModelWriteCore` for model serialization procedure.
+- Upgraded api-version tag from 'package-2023-03-01' to 'package-2023-09-01'. Tag detail available at https://github.com/Azure/azure-rest-api-specs/blob/e19b5771846718774c0452800b9a42c251319f48/specification/vmware/resource-manager/readme.md.
 
 ### Breaking Changes
-
-### Bugs Fixed
-
-### Other Changes
+- WorkloadNetworkResource is now a singleton resource
+- CommonClusterProperties type has been removed
```

Comment:
this does not look correct because we never remove a type and it is not really removed here.
We could just remove this line.